pkg/operation/botanist/component/kubernetesdashboard/kubernetes_dashboard.go

```diff
@@ -0,0 +1,560 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetesdashboard
+
+import (
+	"context"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/autoscaling/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	vpaautoscalingv1 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1"
+	"k8s.io/utils/pointer"
+
+	"github.com/Masterminds/semver"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	resourcesv1alpha1 "github.com/gardener/gardener/pkg/apis/resources/v1alpha1"
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	"github.com/gardener/gardener/pkg/utils/managedresources"
+	"github.com/gardener/gardener/pkg/utils/version"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	name        = "kubernetes-dashboard"
+	scraperName = "dashboard-metrics-scraper"
+	namespace   = "kubernetes-dashboard"
+	labelKey    = "k8s-app"
+	labelValue  = "kubernetes-dashboard"
```

Comment:
We could also extract the logging configuration into `CentralLoggingConfiguration` func - ref https://github.com/gardener/gardener/blob/9b550f3437872a7d10e960119f6e4aaf31bd98c7/charts/seed-bootstrap/charts/fluent-bit/templates/_helpers.tpl#L61-L67 and https://github.com/gardener/gardener/blob/9b550f3437872a7d10e960119f6e4aaf31bd98c7/charts/seed-bootstrap/charts/fluent-bit/templates/_helpers.tpl#L195-L200

Reply:
Update it in a7e7c99381e0773f1b779c793a92b3764cc4e29b. PTAL